test/runtests.jl

```diff
@@ -317,6 +339,30 @@ end
             @test isequal(table_data, data)
             close(result)
 
+            # testing loading to database using CSV.jl row iterator
+            result = execute(
```

Comment:
Rather than delete, you should close the connection and create the temporary table fresh.

Reply:
Done